cypress/integration/page.spec.js

```diff
@@ -1183,18 +1274,26 @@
       page.waitForRequest('@commentsPostRequest');
 
       newCommentForm.nameInput().type('{selectAll}{backspace}Misha Hrynko');
-      newCommentForm.emailInput().type('{selectAll}{backspace}misha@mate.academy');
+      newCommentForm
```

Comment:
Consider using a more descriptive email for the test input to improve readability.